corehq/apps/reports_core/static/reports_core/js/charts.js

```diff
@@ -165,7 +165,7 @@ var charts = (function() {
                 }
                 var $svg = d3.select(chartContainer[0]).append("svg");
                 var id = 'chart-' + i;
-                $svg.attr({id: id, width: "50%", height: "200"});
+                $svg.attr({id: id, height: "400"});
```

Comment:
@benrudolph highly doubtful. wrote this during a hackathon so guessing it was just something that looked better and never came back to it.